ci/helpers/brokerapi_helper.go

```diff
@@ -0,0 +1,308 @@
+package helpers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Does ginkgo need to be included in the helpers?

Reply:
GinkgoWriter is used in GetLastOperationState

Reply:
Ah. Cool. Would it work with `ginkgo.GinkgoWriter`? I'd like to prevent global import if possible.